tests/unit/utils/test_utils_record.py

```diff
@@ -294,3 +300,9 @@ def test_get_value_returns_none_on_index_error():
     })
 
     assert get_value(single_title, 'titles.title[1]') is None
+
+
+def test_get_record_type_returns_none_when_no_type():
+    record = Record({})
+
+    assert get_record_type(record) is None
```

Comment:
Sure, I can do that and the test will fail because the implementation of the function (to be more exact - implementation of the `InspireRecordPidProvider.schema_to_pid_type()` on which it depends) has no knowledge about existing and non-existing schemas - it's a simple url parsing.

It migh have not been clear in my comment - I'd just like to solve the problem which is going to make not-written-yet test fail :smile: .

See [`InspireRecordPidProvider.schema_to_pid_type()` definition](https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/pidstore/providers.py#L87).